package.json

```diff
@@ -54,8 +54,25 @@
     "plugin": true,
     "exec": "{nodeLoader} {path}",
     "config": {
-      "scanChannels": "0x1ffe",
+      "scanChannels": 8190,
       "discoverAttributes": false
+    },
+    "schema": {
+      "type": "object",
+      "required": [
+        "scanChannels",
+        "discoverAttributes"
```

Comment:
I'm going to say that we shouldn't expose discoverAttributes, and only expose scanChannels to be manipulated by the user.

Reply:
I think that would be fine.